osu.Game.Rulesets.Osu/OsuDifficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Game.Rulesets.Osu.Objects;
+
+namespace osu.Game.Rulesets.Osu.OsuDifficulty.Preprocessing
+{
+    public class OsuDifficultyHitObject
+    {
+        /// <summary>
+        /// The current note. Attributes are calculated relative to previous ones.

```

Comment:
Not sure if "note" is a good way to refer to hit objects. I'd just call it hit object. Also what does "current" mean? It'd probably be more specific to mention that this is just the hitobject to which this `OsuDifficultyHitObject` refers to.